CIP-ledger-evolution/README.md

```diff
@@ -0,0 +1,302 @@
+[CBOR](https://www.rfc-editor.org/rfc/rfc7049)
+and specified with
+[CDDL file](https://www.rfc-editor.org/rfc/rfc8610).
+
+The ledger state is also currently also serialized with CBOR,
+but this may change in the future.
+
+### Plutus script context
+
+The ledger and Plutus scripts have a common interface, namely the script context.
+See [CIP-35] for more information.
+Note that CIPs relating to the script context are relevant to both the ledger and to the Plutus CIP categories.
+
+## Specification
+
+This proposal deals only with the types of change listed in
+[Types of change](#types-of-change), all others are out of scope.
+
+### Changes that require a CIP
+
+This proposal recommends that some of the changes listed in "Types of change" (specified below) should:
+
+* Be proposed in a CIP.
+* Go through additional process in addition to the usual CIP process.
+
+The additional process mostly takes the form of additional information that should be present in
+the CIP before it moves to particular stages. As such, it is up to the CIP Editors to enforce this.
+
+The requirement to propose a change via a CIP is, as all CIPs are, advisory.
+In exceptional circumstances or where swift action is required, we expect that changes may still
+be made without following this process. In such circumstances,
+a retrospective CIP SHOULD be made after the fact to record the changes and the rationale for them.
+
+Changes that require a CIP do not have to each be in an individual CIP, they can be included in batches or in other CIPs.
+So, for example, a single CIP could propose multiple new related fields in the transaction.
+
+It is very likely the case that if a change does not require a soft or hard fork,
+it does not require a CIP, except in the case of bugs.
+
+A "bug fix" is a change to behavior where:
+- The implemented behavior does not match the specification; or
+- The specified behavior is clearly wrong (in the judgment of relevant experts)
+
+#### Changes that *do not* require a CIP
+
+* performance improvements
+* bug fixes
+* ledger events
+* changes to the protocol parameter values (this is a governance issue, see [CIP-1694])
+
+### Types of change
+
+- [Ledger features](#ledger-features)
+  - [Ledger-script interface](#ledger-script-interface)
+- [Ledger intra-era switch](#ledger-intra-era-switch)
+- [Ledger protocol](#ledger-protocol)
+- [Ledger eras](#ledger-eras-1)
+- [Serialization](#serialization)
+
+#### Ledger features
+
+This is a very broad category.
+Any conceptual group of changes to the ledger semantics,
+involving a single idea or a group of closely related ideas
+(such as adding, modifying, or removing existing features)
+is a `ledger-feature`.
+This category excludes bug fixes and changes to the [ledger protocol](#ledger-protocol).
+Ledger features are assumed to be included in a future ledger era.
+
+To become active, the ledger feature must be included in an active [ledger era](#ledger-eras-1).
```

Comment:
I think I was a bit off-base here, because you are actually doing what I'm suggesting if we view a ledger era as a kind of release, which I think we should.